### PR TITLE
Update »Datenbanktabellen aktualisieren« via Konsole

### PR DESCRIPTION
**Pull request by fkaminski**
Regarding #594 ...

### CODE REVIEW

**Comment by fritzmg** [comment]
Hm, you have now completely removed any mention of the Contao Install Tool in the command line of `installation/install-contao`. Explaining why and when to use the Install Tool is still important, since you have to use it for Contao 4.4 in any case and in all Contao Versions you still have to use it to import templates for example.

File: docs/dev/reference/commands.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ The options are far more important so let's get to them right away:
 
 {{< version "4.9" >}}
 
-With this command you can execute migrations after an update of Contao or of an extension. 
+With this command you can perform migrations after a new installation, an update of Contao or an extension.
```
Comment (fritzmg) [inline docs/dev/reference/commands.md]:
```suggestion
With this command you can perform database updates and migrations after a new installation or update of Contao or an extension.
```

**Comment by fkaminski** [comment]
@fritzmg - Update your reviews in last commit.

File: docs/manual/installation/install-contao.de.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Sobald der Contao Manager alle Pakete installiert hat, musst du das [Contao-Inst
 aufrufen um die Datenbank zu aktualisieren.
 
 
-## Installation über die Kommandozeile {#installation-ueber-die-kommandozeile}
+## Installation über die Kommandozeile
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.de.md]:
```suggestion
## Installation über die Kommandozeile
```

Why did you remove the anchor? Doesn't this change the anchor? If yes, have you checked any references to this anchor?

File: docs/manual/installation/install-contao.de.md [hunk 3]
```diff
@@ -106,11 +106,40 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
 {{% /notice %}}
 
 
-### Datenbanktabellen aktualisieren
+### Datenbank erstellen
+
+Nach der Installation kannst du die Datenbank Aktualisierung über das [Contao-Installtool](/de/installation/contao-installtool/) 
+durchführen. Ab Contao 4.9 steht dir hierzu auch der folgende Befehl auf der Kommandozeile zur Verfügung:
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.de.md]: This is not quite correct. This sounds like you can create a database in the Contao Install Tool, which is not possible. This needs to go into the next section.

File: docs/manual/installation/install-contao.de.md [hunk 4]
```diff
@@ -106,11 +106,40 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
...
+{{% /notice %}}
+
+
+### Contao Backend-Benutzer erstellen
+
+Im Anschluss kannst du über folgenden Konsolen Befehl einen neuen Backend Benutzer mit Administrator Rechten anlegen:
+
+```bash
+php vendor/bin/contao-console contao:user:create --username=h.lewis --email=hlewis@demo.de --password=meinkennwort --language=de --admin
+
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.de.md]:
```suggestion
php vendor/bin/contao-console contao:user:create --username=h.lewis --email=hlewis@demo.de --password=meinkennwort --language=de --admin
```

File: docs/manual/installation/install-contao.en.md [hunk 5]
```diff
@@ -81,12 +89,42 @@ Example: `example.com`points to the directory `/www/example/web`
...
+{{% /notice %}}
+
+
+### Creating Contao backend users
+
+Afterwards you can create a new backend user with administrator rights using the following console command:
+
+```bash
+php vendor/bin/contao-console contao:user:create --username=h.lewis --email=hlewis@demo.org --password=mypass --language=en --admin
+
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.en.md]:
```suggestion
php vendor/bin/contao-console contao:user:create --username=h.lewis --email=hlewis@demo.org --password=mypass --language=en --admin
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by fkaminski** [comment]
Done. Thanks.

File: docs/manual/installation/install-contao.de.md [hunk 6]
```diff
@@ -83,7 +83,7 @@ php -r "unlink('composer-setup.php');"
 ```
 
 
-### Contao über die Kommandozeile installieren {#contao-ueber-die-kommandozeile-installieren}
+### Contao über die Kommandozeile installieren
```
Comment (netzarbeiter) [inline docs/manual/installation/install-contao.de.md]: Bei Umlauten und Sonderzeichen in einem Titel, setzen wir den Link von Hand:

```suggestion
### Contao über die Kommandozeile installieren {#contao-ueber-die-kommandozeile-installieren}
```

File: docs/manual/installation/install-contao.de.md [hunk 7]
```diff
@@ -108,9 +108,36 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
 
 ### Datenbanktabellen aktualisieren
 
-Nach der Installation ist vor der Datenbank-Aktualisierung, rufe dazu das [Contao-Installtool](../contao-installtool/)
-auf oder verwende (ab Contao 4.9) 
+Nach der Installation kannst du die Datenbank Aktualisierung über das [Contao-Installtool](/de/installation/contao-installtool/)
```
Comment (netzarbeiter) [inline docs/manual/installation/install-contao.de.md]:
```suggestion
Nach der Installation kannst du die Datenbank-Aktualisierung über das [Contao-Installtool](/de/installation/contao-installtool/) 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko.

**Review by netzarbeiter — APPROVED** [review]

File: docs/manual/installation/install-contao.de.md [hunk 8]
```diff
@@ -108,9 +108,35 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
...
+{{% /notice %}}
+
+
+### Contao Backend-Benutzer erstellen
+
+Im Anschluss kannst du über folgenden Konsolen Befehl einen neuen Backend Benutzer mit Administrator Rechten anlegen:
+
+```bash
+php vendor/bin/contao-console contao:user:create --username=h.lewis --email=hlewis@demo.de --password=meinkennwort --language=de --admin
+```
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.de.md]: You need to mention the Contao Install Tool here as well, just as in the previous section. And you need to mention that the command is only available in Contao 4.10 and later.

File: docs/manual/installation/install-contao.de.md [hunk 9]
```diff
@@ -108,9 +108,35 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
...
+[hier](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration).
+{{% /notice %}}
+
+
+### Contao Backend-Benutzer erstellen
+
+Im Anschluss kannst du über folgenden Konsolen Befehl einen neuen Backend Benutzer mit Administrator Rechten anlegen:
+
+```bash
+php vendor/bin/contao-console contao:user:create --username=h.lewis --email=hlewis@demo.de --password=meinkennwort --language=de --admin
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.de.md]:
```suggestion
php vendor/bin/contao-console contao:user:create
```

The command is interactive and will ask you for any information that you did not give via parameters. Also for security reasons you should never set the password via a parameter directly, otherwise the password will be visible in the bash history.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/installation/install-contao.de.md (hunk 8 above)
Comment (fkaminski) [reply docs/manual/installation/install-contao.de.md]: Thank you - See last commit.

File: docs/manual/installation/install-contao.de.md (hunk 9 above)
Comment (fkaminski) [reply docs/manual/installation/install-contao.de.md]: Thank you - See last commit.

File: docs/manual/installation/install-contao.en.md [hunk 10]
```diff
@@ -81,12 +89,40 @@ Example: `example.com`points to the directory `/www/example/web`
...
+{{% notice tip %}}
+You can also create a database beforehand on the command line:<br>
+`php vendor/bin/contao-console doctrine:database:create`
+{{% /notice %}}
+
+{{% notice info %}}
+Contao needs to know the corresponding connection data for your database. This information can either be retrieved via 
+an existing "config/paramters.yml" (currently installed using the [Contao-Install tool](/en/installation/contao-installtool/) 
+) or via a "[.env](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration)" file
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.en.md]:
```suggestion
an existing "config/paramters.yml" (currently installed using the [Contao-Install tool](/en/installation/contao-installtool/))
or via a "[.env](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration)" file 
```

File: docs/manual/installation/install-contao.en.md [hunk 11]
```diff
@@ -81,12 +89,40 @@ Example: `example.com`points to the directory `/www/example/web`
...
+For details on the necessary environment variables ([DATABASE_URL](https://docs.contao.org/dev/reference/config/#database-url) 
+and [APP_SECRET](https://docs.contao.org/dev/reference/config/#app-secret)) in a ".env" file you can find 
+[here](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration).
+{{% /notice %}}
+
+
+### Creating Contao backend users
+
+Using the [Contao-Installtool](/en/installation/contao-installtool/), you can create your backend user. Since Contao 
+4.10.x you can use the following command on the command line:
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.en.md]:
```suggestion
### Creating Contao back end users

Using the [Contao-Installtool](/en/installation/contao-installtool/), you can create your back end user. Since Contao 
**4.10** you can use the following command on the command line:
```

File: docs/manual/installation/install-contao.de.md [hunk 12]
```diff
@@ -108,9 +108,37 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
...
+Weitere Details für die notwendigen Umgebungsvariablen ([DATABASE_URL](https://docs.contao.org/dev/reference/config/#database-url) 
+und [APP_SECRET](https://docs.contao.org/dev/reference/config/#app-secret)) in einer ».env« Datei findest du 
+[hier](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration).
+{{% /notice %}}
+
+
+### Contao Backend-Benutzer erstellen
+
+Über das [Contao-Installtool](/de/installation/contao-installtool/) kannst du dir deinen Backend Benutzer anlegen. 
+Ab Contao 4.10.x kannst du hierzu den folgenden Befehl auf der Kommandozeile benutzen. Die notwendigen Angaben werden dann
```
Comment (fritzmg) [inline docs/manual/installation/install-contao.de.md]:
```suggestion
Ab Contao **4.10** kannst du hierzu den folgenden Befehl auf der Kommandozeile benutzen. Die notwendigen Angaben werden dann
```

Comment (fkaminski) [reply docs/manual/installation/install-contao.de.md]: See last commit

**Review by fritzmg — APPROVED** [review]
